src/fsharp/Fsc-proto/Fsc-proto.fsproj

```diff
@@ -1,493 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <FSharpSourcesRoot>$(MSBuildProjectDirectory)\..\..</FSharpSourcesRoot>
-    <Configuration Condition=" '$(Configuration)' == '' ">Proto</Configuration>
-  </PropertyGroup>
-  <Import Project="$(FSharpSourcesRoot)\FSharpSource.Settings.targets" />
-  <PropertyGroup>
-    <PlatformTarget>x86</PlatformTarget>
-    <OutputType>Exe</OutputType>
-    <NoWarn>$(NoWarn);62</NoWarn>
-    <AssemblyName>fsc</AssemblyName>
-    <DefineConstants>NO_EXTENSIONTYPING;$(DefineConstants)</DefineConstants>
```

Comment:
NO_EXTENSIONTYPING is normally set for Proto, since type providers aren't used by the compiler itself.  I would like to keep that - could you make sure that when using FSharp.Compiler.Private.fsproj we keep that?